Log4swift/Utilities/String+utilities.swift

```diff
@@ -71,9 +71,31 @@ extension String {
     
     return str as String
   }
+
+
+  /// Returns a dictionary if String contains proper JSON format for a single, non-nested object; a simple dictionary.
+  /// Keys and values should be surrounded with single or double quotes.
+  /// Ex: {"name":"value", 'name':'value'}
+  public func toDictionary() -> [String:AnyObject] {
+    var dict = [:]
+
+    let s = (self as NSString).stringByReplacingOccurrencesOfString("'", withString: "\"")
+
+    if let data = s.dataUsingEncoding(NSUTF8StringEncoding) {
+      do {
+        dict = try NSJSONSerialization.JSONObjectWithData(data, options: NSJSONReadingOptions.AllowFragments) as! [String:AnyObject]
+      }
+      catch {
+        // TODO: Report invalid format somehow?
```

Comment:
Silent errors should really be avoided.
I suggest to not catch this error, and let the caller handle it. In the current case, if the parameters are decoded when reading the pattern, at initialization time, which is already handled.